src/components/ManagerPage/FirmwareUpdate.js

```diff
@@ -102,27 +102,26 @@ class FirmwareUpdate extends PureComponent<Props, State> {
     invariant(latestFirmware, 'did not find a new firmware or firmware is not set')
 
     this.setState({ modal: 'install' })
-    const { success } = await installOsuFirmware
+    const result = await installOsuFirmware
       .send({ devicePath: device.path, firmware: latestFirmware, targetId: deviceInfo.targetId })
       .toPromise()
-    return success
-  }
 
-  installFinalFirmware = async (device: Device) => {
-    const { success } = await installFinalFirmware.send({ devicePath: device.path }).toPromise()
-    return success
+    return result
```

Comment:
it looks like you used to return `success` but now you return `result` 🤔  
i guess you don't use the result value anyway?